ui/app/AppLayouts/Profile/panels/ProfileDescriptionPanel.qml

```diff
@@ -53,7 +53,12 @@ Item {
             minimumHeight: 108
             maximumHeight: 108
             input.verticalAlignment: TextEdit.AlignTop
-
+            validators: [
+                StatusRegularExpressionValidator {
+                    regularExpression: Constants.regularExpressions.asciiWithEmoji
+                    errorMessage: qsTr("Only letters, numbers, ASCII characters or emojis allowed")
```

Comment:
The error message has been updated in Figma to remove the ASCII reference.
https://www.figma.com/design/ibJOTPlNtIxESwS96vJb06/👤-Profile-|-Desktop?node-id=2460-8755&t=7EUM0YB7om3UmscW-0